wallet/lib/pages/detailed_asset_view/owner_view.dart

```diff
@@ -80,6 +82,8 @@ class OwnerViewContent extends StatefulWidget {
 }
 
 class _OwnerViewContentState extends State<OwnerViewContent> {
+  bool showLoader = true;
```

Comment:
The loader should be a part of a 3d widget. Owner view already have video , image and audio functionalities.